client/app/telecom/pack/xdsl/missingRio/pack-xdsl-missing-rio.html

```diff
@@ -0,0 +1,91 @@
+                                                   data-ng-value="false"
+                                                   name="xdslPortability"
+                                                   required />
+                                            <span data-translate="xdsl_missing-rio_port-number_label"
+                                                  data-translate-values="{ num: PackXdslMissingRio.number }"></span>
+                                        </label>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="form-group"
+                                 data-ng-if="!PackXdslMissingRio.missingRioForm.portNumber"
+                                 data-ng-class="{
+                                     'has-error': missingRioForm.rio.$touched && missingRioForm.rio.$invalid
+                                 }">
+                                <label for="rio"
+                                       class="control-label d-inline-block"
+                                       data-tooltip-placement="right"
+                                       data-tooltip-append-to-body="true"
+                                       data-uib-tooltip="{{ ::'xdsl_missing-rio_label_tooltip' | translate: {num: PackXdslMissingRio.number} }}">
+                                    <span data-translate="xdsl_missing-rio_label"></span>
+                                    <i class="ovh-font ovh-font-question ml-1"
+                                       aria-hidden="true"></i>
+                                </label>
+                                <input id="rio"
+                                       name="rio"
+                                       class="form-control"
+                                       data-ng-model="PackXdslMissingRio.missingRioForm.rio"
+                                       maxlength="12"
+                                       required
+                                       type="text" />
+                                <span class="help-block"
+                                      role="alert"
+                                      data-ng-if="missingRioForm.rio.$touched"
+                                      data-ng-messages="missingRioForm.rio.$error">
+                                    <span data-ng-message="required"
+                                          data-translate="xdsl_missing-rio_required">
+                                    </span>
+                                </span>
+                            </div>
+                            <button
+                                type="submit"
+                                data-ng-disabled="missingRioForm.$invalid"
+                                class="btn btn-primary mt-2">
+                                <span data-translate="xdsl_missing-rio_send"></span>
```

Comment:
Apply `data-translate="xdsl_missing-rio_send"` directly on the `button` element.